plugins/modules/cobbler_system.py

```diff
@@ -278,7 +278,7 @@ def main():
         if system:
             # Update existing entry
             system_id = ""
-            if LooseVersion(str(conn.version())) >= LooseVersion("3.4"):
+            if LooseVersion(str(conn.version())) >= LooseVersion("3.400"):
```

Comment:
But in this case your solution isn't correct either, since if for `3.4.0`, `conn.version()` will be `3.4`, and not `3.400`. You need to compare versions as floats then:
```suggestion
            # https://github.com/cobbler/cobbler/blame/6035974056a9df61d273601ba49b55ef19319440/cobbler/api.py#L234
            if float(conn.version()) >= 3.4:
```